modules/node_modules/@frogpond/ccci-stolaf-college/v1/jobs/index.mjs

```diff
@@ -6,55 +6,75 @@ import getUrls from 'get-urls'
 const {JSDOM} = _jsdom
 
 export function cleanJob(job) {
-	const title = JSDOM.fragment(job.title).textContent.trim()
-	const office = JSDOM.fragment(job.office).textContent.trim()
-	const type = JSDOM.fragment(job.type).textContent.trim()
+	const id = job.id
```

Comment:
For single-line things like this, can you just stick them into the result directly?

instead of `const id = job.id; return {id}`, just do `[…]; return {id: job.id}`?